main_FractureDynamics.C

```diff
@@ -355,7 +349,12 @@ int runSimulator (const FractureArgs& args)
   case 2:
     return runSimulator1<Dim,GenAlphaSIM>(args);
   case 3:
-    return runSimulator1<Dim,NonLinSIM>(args);
+    if (args.coupling == 2)
+      return runSimulator1<Dim,NonLinSIM,QstaticFracture>(args);
+    else if (args.coupling == 3)
+      return runSimulator1<Dim,NonLinSIM,MieheFracture>(args);
+    else
+      return runSimulator1<Dim,NonLinSIM>(args);
```

Comment:
consider `switch` - `case` instead of `if` - `else if` - `else` also here?

Reply:
switched.